feature-libs/product-configurator/rulebased/components/overview-form/configurator-overview-form.component.html

```diff
@@ -35,17 +35,16 @@ <h2>{{ 'configurator.overviewForm.noAttributeHeader' | cxTranslate }}</h2>
       id="{{ getGroupId(idPrefix, group.id) }}"
       [ngClass]="getGroupLevelStyleClasses(level, group.subGroups)"
     >
-      <span class="cx-visually-hidden">
-        {{
-          'configurator.a11y.group'
-            | cxTranslate
-              : {
-                  group: group.groupDescription,
-                }
-        }}
-      </span>
-      <h2 aria-hidden="true">
-        <span>{{ group.groupDescription }}</span>
+      <h2>
```

Comment:
That's breaking, we need to introduce a feature flag here

Reply:
Added feature toggle